lib/classes/Variables.js

```diff
@@ -476,13 +476,14 @@ class Variables {
   splitByComma(string) {
     const input = string.trim();
     const stringMatches = [];
-    let match = this.stringRefSyntax.exec(input);
+    const regex = /(?:('|").*?\1)/g;
```

Comment:
I had to copy the original regexp to here because it was being reused. Here it's actually the correct regexp as far as I can tell.